src/DependencyInjection/Configuration.php

```diff
@@ -15,33 +15,71 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode = $treeBuilder->getRootNode();
 
-        $rootNode->children()
-            ->arrayNode('health')
-            ->canBeDisabled()
+        $rootNode
             ->children()
-            ->arrayNode('builtin')
-            ->addDefaultsIfNotSet()
-            ->children()
-            ->arrayNode('disk_space')
-            ->addDefaultsIfNotSet()
-            ->canBeDisabled()
-            ->children()
-            ->integerNode('threshold')->defaultValue(50 * 1024 * 1024)->end()
-            ->scalarNode('path')->defaultValue('%kernel.project_dir%')->end()
-            ->end()
-            ->end()
-            ->end()
-            ->end()
-            ->end()
-            ->end()
-            ->arrayNode('info')
-            ->addDefaultsIfNotSet()
-            ->canBeDisabled()
-            ->children()
-            ->arrayNode('builtin')->defaultValue(['php', 'symfony', 'git'])->scalarPrototype()->validate()->ifNotInArray(['php', 'symfony', 'git'])->thenInvalid('Invalid info builtin configuration. Available builtins are: php, symfony and git')->end()
-            ->end()
-            ->end()
-            ->end()
+                ->arrayNode('health')
+                    ->canBeDisabled()
+                    ->children()
+                        ->arrayNode('builtin')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->arrayNode('disk_space')
+                                    ->addDefaultsIfNotSet()
+                                    ->canBeDisabled()
+                                    ->children()
+                                        ->integerNode('threshold')->defaultValue(50 * 1024 * 1024)->end()
+                                        ->scalarNode('path')->defaultValue('%kernel.project_dir%')->end()
+                                    ->end()
+                                ->end()
+                                ->arrayNode('database')
+                                    ->addDefaultsIfNotSet()
+                                    ->canBeDisabled()
+                                    ->children()
+                                        ->arrayNode('connections')
+                                            ->useAttributeAsKey('name')
+                                            ->defaultValue(['default' => ['service' => 'Doctrine\DBAL\Connection', 'check_sql' => 'SELECT 1']])
```

Comment:
can we add this to readme also (with some examples?)

Reply:
I've updated the `README` and outlined how you can specify the connection service aliases. I hope you find it sufficient. I would like to propose to update the README if we add more components (i'm planning to add the mailer and messager component soon)

Reply:
:heart_eyes: